### PR TITLE
CI: Update coverage

### PR DESCRIPTION
**Pull request by effigies**

### CODE REVIEW

File: .codecov.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ignore:
+  - "fitlins/_version.py"
```
Comment (effigies) [inline .codecov.yml]:
```suggestion
  - "smriprep/_version.py"
```

File: setup.cfg [hunk 2]
```diff
@@ -112,3 +112,12 @@ filterwarnings =
 
 [coverage:run]
 branch = True
+source = smriprep
+omit =
+    */tests/*
+relative_files = True
+
+[paths]
```
Comment (effigies) [inline setup.cfg]:
```suggestion
[coverage:paths]
```

**Comment by effigies** [comment]
Well, everything works, except that the coverage report shows:

```
Name                                  Stmts   Miss Branch BrPart  Cover
-----------------------------------------------------------------------
smriprep/__about__.py                     8      8      0      0     0%
smriprep/__init__.py                      3      3      0      0     0%
smriprep/__main__.py                      6      6      4      0     0%
smriprep/_version.py                    282    282    146      0     0%
smriprep/cli/__init__.py                  0      0      0      0   100%
smriprep/cli/run.py                     226    226     56      0     0%
smriprep/cli/utils.py                    31     31     12      0     0%
smriprep/conf/__init__.py                 3      3      0      0     0%
smriprep/interfaces/__init__.py           4      4      0      0     0%
smriprep/interfaces/freesurfer.py        65     65     41      0     0%
smriprep/interfaces/reports.py           75     75      8      0     0%
smriprep/interfaces/templateflow.py      34     34      4      0     0%
smriprep/utils/__init__.py                0      0      0      0   100%
smriprep/utils/bids.py                   30     30     12      0     0%
smriprep/workflows/__init__.py            0      0      0      0   100%
smriprep/workflows/anatomical.py         71     71     12      0     0%
smriprep/workflows/base.py               55     55     18      0     0%
smriprep/workflows/norm.py               38     38      8      0     0%
smriprep/workflows/outputs.py           103    103     14      0     0%
smriprep/workflows/surfaces.py           90     90     20      0     0%
-----------------------------------------------------------------------
TOTAL                                  1124   1124    355      0     0%
```

**Comment by effigies** [comment]
Yeah, I think the codecov config is fine. Need to fiddle more with coverage itself to figure out why it's not picking up anything.

**Comment by oesteban** [comment]
Should we merge and add your fiddling in another PR?

**Comment by effigies** [comment]
I'm not really sure what value this PR adds if the coverage is always 0%.

I'd say let's get it working before we merge.

Sorry for the premature request for review. It seemed to be working at the time.

**Pull request merged**